### PR TITLE
Separate class responsibilities

### PR DESCRIPTION
**Pull request by jcalarcon98**
# Birthday Slack Bot

## Authors

- Jean Carlos Alarcón @jcalarcon98 

## Description

This PR adds Slack API integration to get Slack user information based on bamboo email in order to tag a user in the message.

### CODE REVIEW

File: src/integrations/slack.py [hunk 1]
```diff
@@ -1,28 +1,55 @@
-from ..clients.rest import RestClient
-from ..config import EnvManager
+from typing import List
+
+from src.clients import RestClient
+from src.config import EnvManager
 
 
 class SlackIntegration:
```
Comment (Wason1797) [inline src/integrations/slack.py]: Maybe we can rename this class to SlackBotIntegration or SlackApiIntegration

File: src/controllers/birthday_message.py [hunk 2]
```diff
@@ -44,15 +44,16 @@ def is_birthday(birthday: str):
         return [user.get(BambooIntegration.employee_identifier_field) for user in users if is_birthday(user.get(BambooIntegration.employee_birthday_field))]
```
Comment (Wason1797) [inline src/controllers/birthday_message.py]: maybe we can check if birthday is not None, just to make sure

Comment (Wason1797) [inline src/controllers/birthday_message.py]: Maybe we can separate the mapping logic to another function in the BambooIntegration class

**Review by Wason1797 — CHANGES_REQUESTED** [review]
Some minor things to check

File: src/integrations/slack.py (hunk 1 above)
Comment (jcalarcon98) [reply src/integrations/slack.py]: Done!

File: src/controllers/birthday_message.py (hunk 2 above)
Comment (jcalarcon98) [reply src/controllers/birthday_message.py]: Great, suggestion applied!

Comment (jcalarcon98) [reply src/controllers/birthday_message.py]: Done!

**Review by Wason1797 — APPROVED** [review]
Nice!!

**Pull request merged**